tests/beaker_tests/cisco_stp_global/test_stp_global.rb

```diff
@@ -280,6 +286,8 @@ def test_harness_dependencies(_tests, id)
   test_harness_run(tests, :non_default_bd)
   test_harness_run(tests, :non_default_plat_1)
   test_harness_run(tests, :non_default_plat_2)
+
+  test_cleanup_all_bridge_domains
```

Comment:
yes it does that in line 268 already.  Probably there should be a guard for n7k product-id for this function.